modules/silicon_design/scripts/build/notebooks/examples/renode/renode.md

```diff
@@ -0,0 +1,195 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+## Renode run
+
+
+```python tags=["parameters"]
+quantum = 0.001
+mbs = 10000
+mips = 100
+```
+
+
+```python
+! pip install -q git+https://github.com/antmicro/renode-colab-tools.git # only needed in the Colab environment
```

Comment:
add those to the environment instead?

Reply:
Sure, sounds like a good idea! We kept it here as Renode is not the main focus of the `silicon_design` notebook but I think it will be easier if it's just an implicit requirement.

Reply:
Yep, I don't mind adding new requirements that are notebook specific to the base image if it makes them easier to run and read.